lib/standalone.cjs

```diff
@@ -1,29 +1,29 @@
 'use strict';
 
+const node_path = require('node:path');
+const node_fs = require('node:fs');
 const process = require('node:process');
-
-const debug = require('debug')('stylelint:standalone');
+const createDebug = require('debug');
 const fastGlob = require('fast-glob');
-const fs = require('fs');
 const globby = require('globby');
 const normalizePath = require('normalize-path');
-const path = require('path');
-
-const createStylelint = require('./createStylelint.cjs');
+const writeFileAtomic = require('write-file-atomic');
+const allFilesIgnoredError = require('./utils/allFilesIgnoredError.cjs');
+const noFilesFoundError = require('./utils/noFilesFoundError.cjs');
+const validateTypes = require('./utils/validateTypes.cjs');
 const createPartialStylelintResult = require('./createPartialStylelintResult.cjs');
+const createStylelint = require('./createStylelint.cjs');
 const filterFilePaths = require('./utils/filterFilePaths.cjs');
-const formatters = require('./formatters/index.cjs');
+const index = require('./formatters/index.cjs');
```

Comment:
nit

Why the rename?

Reply:
`.cjs` files are automatically generated by Rollup. So I don't know why. 😅

Reply:
I mean the const name.
Why rename it index when you kept formatters in the mjs counterpart?

Reply:
I don't touch the const name. Rollup determines the name from the `.mjs` file, but I don't know why Rollup renamed it.

Reply:
Just to chime in - I think it's because the file is named `index`?